.env.example

```diff
@@ -2,13 +2,14 @@
 
 # Important variables
 BASE_URL=localhost.test
-
 HOSTNAME=
-EMAIL_DOMAIN=
 
 #we wish :(
 SHORTENER_DOMAIN=aeg.ee
 
+EMAIL_DOMAIN=
```

Comment:
what is this used for btw?

Reply:
because BASE_URL will not always be our domain (especially now that i am making all the tests with barnabeu)